pages/zap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Using OWASP ZAP
+permalink: /zap/
+---
+
+# The OWASP Zed Attack Proxy (ZAP)
+
+[OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is a tool for doing automated vulnerability scanning.
+
+## Usage
+
+You will need a running application to test. If testing a live URL, make sure to let your team know, as they may see a spike in requests and errors.
+
+1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
+1. Start your application.
+    * If using Rails on Mac, start it with `rails server --binding 127.0.0.1` (presumably due to [this change](http://guides.rubyonrails.org/4_2_release_notes.html#default-host-for-rails-server)).
```

Comment:
Kinda beside the point, but ran through some tests for posterity:

binding | attack URL | works?
--- | --- | ---
none | http://localhost:3000 | N
none | http://127.0.0.1:3000 | Y
none | http://0.0.0.0:3000 | N
`127.0.0.1` | http://localhost:3000 | Y
`127.0.0.1` | http://127.0.0.1:3000 | Y
`127.0.0.1` | http://0.0.0.0:3000 | N
`0.0.0.0` | http://localhost:3000 | Y
`0.0.0.0` | http://127.0.0.1:3000 | Y
`0.0.0.0` | http://0.0.0.0:3000 | Y